test/case/03_test_update_from_master

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -x
+
+HERE=${0%/*}
+
+REPO_ROOT=$(git rev-parse --show-toplevel)
+
+source $REPO_ROOT/test/lib.sh
+
+repo=$(empty_repo)
+add_cleanup $repo
+
+bootstrap_repo $repo
+
+cd $repo
+
+# TODO: test REPO_NAME detection by setting an appropriately-formatted
+# git remote
+export REPO_NAME=${0##*/}
+
+override_boilerplate_version $(new_boilerplate_repo)/boilerplate
+
+make update_boilerplate
+
+check_update 
+
+if [ $? -ne 0 ] ; then
+	exit $?
+fi
+
+add_convention test/test_base_convention
```

Comment:
For now, it is failing, pending https://github.com/openshift/boilerplate/pull/16 to be merged